master/buildbot/reporters/telegram.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import absolute_import
```

Comment:
do we need these two? master [expects](http://docs.buildbot.net/current/manual/installation/requirements.html) a reasonable version of Python 3.